sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -660,6 +690,17 @@ Write withBigtableService(BigtableService bigtableService) {
       return toBuilder().setBigtableConfig(config.withBigtableService(bigtableService)).build();
     }
 
+    /**
+     * Returns a {@link BigtableIO.WriteWithResults} that will emit a {@link BigtableWriteResult}
+     * for each batch of rows written.
+     *
+     * <p>Does not modify this object.
```

Comment:
"Does not modify this object" is redundant, it is true for all methods of all builders in Beam.

Reply:
🤷‍♂ I was just being consistent with all the other methods in this class.